pages/nfttest/mint/_iscnId.vue

```diff
@@ -172,18 +181,35 @@ export default class NFTTestMintPage extends Vue {
   }
 
   async mounted() {
-    await this.getMintInfo()
+    this.getMintInfo()
     await this.getISCNInfo()
```

Comment:
Please use Promise.all if they are all async but we don't need them to be blocking